### ISSUE TITLE
False Positive on taglibs-standard-impl-1.2.5.jar  

### ISSUE THREAD

**Issue by Anshu2405** [dependency-check/DependencyCheck#3191]
False positive on library taglibs-standard-impl-1.2.5.jar   - reported as cpe:2.3:a:tag_project:tag:*:*:*:*:*:*:*:*
```xml
<dependency>
   <groupId>org.apache.taglibs</groupId>
    <artifactId>taglibs-standard-impl</artifactId>
    <version>1.2.5</version>
</dependency>
```

**Comment by Anshu2405** [comment]
Hi ,

Can we know the status if the CVEs are considered as false positive ?

**Comment by aikebah** [comment]
Looks like you use an outdated version of DependencyCheck. FP should be fixed since 6.0.5
